assets/src/components/mapMarkers.tsx

```diff
@@ -135,7 +157,19 @@ export const VehicleMarker = ({
         icon={vehicleIcon}
         eventHandlers={eventHandlers}
         zIndexOffset={zIndexOffset}
-      />
+        ref={markerRef}
+      >
+        {inTestGroup(TestGroups.DetoursPilot) && (
+          <Popup className="c-dropdown-popup-wrapper" offset={[140, 97]}>
```

Comment:
**suggestion:** I think per some of our discussion having the hardcoded offset is somewhat inevitable, but if it's possible I think a quick comment explaining how this particular value was derived would be helpful, especially if we go mess with the styling in the future and the current offset no longer looks good.

Reply:
https://github.com/mbta/skate/pull/2373/commits/6e682ea8fa4b97d1b319f6b60e6378cd2f8b6628

Happy to keep iterating on the wording in ☝️ comment - this is just a first draft.

Reply:
I think that's fine, but it would also be nice if we could say something more specific like "this was derived from taking the rendered size of the `.c-foo` element and adding `X` pixels of padding to it" or whatever you did. Or maybe I'm imagining something more systematic when it's more a result of experimenting until it looked good.

Reply:
I experimented until it looked good 😅

Reply:
Heh okay totally valid! In that case I think this comment is fine.